pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/fwd/SingleValueSortedForwardIndexCreator.java

```diff
@@ -56,10 +56,13 @@ public void index(int docId, int dictId) {
   public void close()
       throws IOException {
     int cardinality = _maxDocIds.length;
-    for (int i = 0; i < cardinality; i++) {
-      _writer.setInt(i, 0, _minDocIds[i]);
-      _writer.setInt(i, 1, _maxDocIds[i]);
+    try {
```

Comment:
Use try with resource

Reply:
May be I am missing something but how can we use try with resource here? It can only be used if the resource is being declared/created there for immediate usage and released thereafter soon after the try block ends. In this case, we are dealing with resources that are member variables.

Reply:
```
try (Closeable closeable = _writer) {
  ...
}
```

Reply:
Got it. Made the change